src/v3/src/components/StepperButton/StepperButton.tsx

```diff
@@ -46,12 +49,13 @@ const StepperButton: UISchemaElementComponent<{
     <Button
       onClick={handleClick}
       variant={variant ?? 'primary'}
-      type="button"
+      type={type ?? 'button'}
       fullWidth
       ref={focusRef}
       sx={{
         whiteSpace: 'normal',
       }}
+      startIcon={loading && <Spinner color="white" />}
```

Comment:
Had to add support for`type=submit` and a `Spinner` icon in this component to make it replicate the behavior of a normal submit `Button` since we need submit button functionality within a `StepperLayout` for this flow